src/legacy/core_plugins/embeddable_api/public/panel/_embeddable_panel.scss

```
@@ -0,0 +1,134 @@
+.embPanel {
+  z-index: auto;
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  min-height: $euiSizeL + 2px; // + 2px to account for border
+  position: relative;
+
+  // SASSTODO: The inheritence factor stemming from embeddables makes this class hard to change
+  .embPanel__content {
+    display: flex;
+    flex: 1 1 100%;
+    z-index: 1;
+    min-height: 0; // Absolute must for Firefox to scroll contents
+  }
+
+  // SASSTODO: Pretty sure this doesn't do anything since the flex-basis 100%,
+  // but it MIGHT be fixing IE
+  .embPanel__content--fullWidth {
+    width: 100%;
+  }
+
+  .embPanel__content-isLoading {
+    // completely center the loading indicator
+    justify-content: center;
+    align-items: center;
+  }
+}
+
+// HEADER
+
+.embPanel__header {
+  flex: 0 0 auto;
+  display: flex;
+  // ensure menu button is on the right even if the title doesn't exist
+  justify-content: flex-end;
+}
+
+.embPanel__title {
+  @include euiTextTruncate;
+  @include euiTitle('xxxs');
+  line-height: 1.5;
+  flex-grow: 1;
+
+  &:not(:empty) {
+    padding: ($euiSizeXS * 1.5) $euiSizeS 0;
+  }
+}
+
+.embPanel__dragger:not(.embPanel__title) {
+  flex-grow: 1;
+}
+
+.embPanel__header--floater {
+  position: absolute;
+  right: 0;
+  top: 0;
+  left: 0;
+  z-index: $euiZLevel1;
+}
+
+// OPTIONS MENU
+
+/**
+  * 1. Use opacity to make this element accessible to screen readers and keyboard.
+  * 2. Show on focus to enable keyboard accessibility.
+  * 3. Always show in editing mode
+  */
+
+.embPanel__optionsMenuButton {
+  background-color: transparentize($euiColorDarkestShade, .9);
+  border-bottom-right-radius: 0;
+  border-top-left-radius: 0;
+
+  &:focus {
+    background-color: $euiFocusBackgroundColor;
+  }
+}
+
+.embPanel
+.embPanel__optionsMenuButton {
+  opacity: 0; /* 1 */
+
+  &:focus {
+    opacity: 1; /* 2 */
+  }
+}
+
+.embPanel__optionsMenuPopover[class*="-isOpen"],
+.embPanel:hover  {
+  .embPanel__optionsMenuButton {
+    opacity: 1;
+  }
+}
+
+// EDITING MODE
+
+.embPanel--editing {
+  border-style: dashed;
+  border-color: $euiColorMediumShade;
+  transition: all $euiAnimSpeedFast $euiAnimSlightResistance;
+
+  &:hover,
+  &:focus {
+    @include euiSlightShadowHover;
+  }
+
+  .embPanel__dragger {
+    transition: background-color $euiAnimSpeedFast $euiAnimSlightResistance;
+
+    &:hover {
+      background-color: $embEditingModeHoverColor;
+      cursor: move;
+    }
+  }
+
+  .embPanel__optionsMenuButton {
+    opacity: 1; /* 3 */
+  }
+}
+
+.embPanel__error {
+  text-align: center;
+  justify-content: center;
+  flex-direction: column;
+  overflow: auto;
+  text-align: center;
```

Can you add some padding to the error?:
```suggestion
  text-align: center;
  padding: $euiSizeS;
```